cargo-guppy/src/main.rs

```diff
@@ -26,6 +27,37 @@ enum Command {
     #[structopt(name = "dups")]
     /// Print the number of duplicate packages
     Duplicates,
+    #[structopt(name = "select")]
+    /// Select packages and their transitive dependencies
+    Select {
+        #[structopt(long, short, possible_values = &Kind::variants(), case_insensitive = true, default_value = "all")]
+        /// Kind of crates to select
+        kind: Kind,
+        #[structopt(long, rename_all = "kebab-case")]
+        /// Include dev dependencies
+        include_dev: bool,
+        #[structopt(long, rename_all = "kebab-case")]
+        /// Include build dependencies
+        include_build: bool,
+        #[structopt(long)]
+        /// Target to select for, default is to match all targets
+        target: Option<String>,
+        #[structopt(long)]
+        /// Save selection graph in .dot format
+        output: Option<String>,
```

Comment:
Hmm, maybe this arg should be called `output-dot` or similar?

Reply:
Updated.